xml/System.Windows.Media/RadialGradientBrush.xml

```diff
@@ -33,7 +33,7 @@ Radial gradient with a highlighted focal point
 >  <xref:System.Windows.Media.RadialGradientBrush> objects are rendered using hardware acceleration on <xref:System.Windows.Media.RenderCapability.Tier%2A> 2 systems. For more information about hardware tiers, see [Graphics Rendering Tiers](~/docs/framework/wpf/advanced/graphics-rendering-tiers.md).  
   
 ## Freezable Features  
- Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.RadialGradientBrush> class provides several special features: <xref:System.Windows.Media.RadialGradientBrush> objects can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.RadialGradientBrush> class provides several special features: <xref:System.Windows.Media.RadialGradientBrush> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.RadialGradientBrush> class provides several special features: <xref:System.Windows.Media.RadialGradientBrush> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```